src/qt/addresstablemodel.cpp

```diff
@@ -85,7 +86,7 @@ class AddressTablePriv
             BOOST_FOREACH(const PAIRTYPE(CTxDestination, CAddressBookData)& item, wallet->mapAddressBook)
             {
                 const CBitcoinAddress& address = item.first;
-                bool fMine = IsMine(*wallet, address.Get());
+                bool fMine = wallet->IsMine(address.Get());
```

Comment:
This calls an IsMine() overload within wallet.cpp::989 which calls the IsMine() within wallet_ismine.cpp passing chainActive.Tip() as the CBlockindex.